src/libraries/System.Runtime.InteropServices/tests/System.Runtime.InteropServices.UnitTests/System/Runtime/InteropServices/Marshal/GetEndComSlotTests.cs

```diff
@@ -70,9 +67,8 @@ public static IEnumerable<object[]> GetStartComSlot_NotComVisibleType_TestData()
             yield return new object[] { collectibleType };
```

Comment:
Same question about `IsReflectionEmitSupported`

Reply:
Yes, they could use that as well. However, this is blocked because it is only used when `IsBuiltInComEnabled` is true. In practice these are orthogonal features, but they tend to come together. This is simple to add if the features become disjoint, but I don't think they are needed right now. Fair?